sjb/config/test_cases/ami_build_origin_int_rhel_fork.yml

```diff
@@ -0,0 +1,223 @@
+---
+parameters:
+- name: "TRELLO_URL"
+  description: "The resulting ami will be named to reference the specified card and a comment will be added to the corresponding trello card when the ami completes.
+If you don't have a corresponding trello card.  You can use anything that will get added to the ami name => fork_ami_openshift3_${TRELLO_URL}_${BUILD_NUMBER}"
+- name: "ORIGIN_FORK_ID"
+  description: "The fork id to use for the origin repo."
+  default_value: "openshift"
+- name: "ORIGIN_BRANCH"
+  description: "Branch to use for origin fork."
+  default_value: "master"
+- name: "ORIGIN_REPO"
+  description: "The name of the origin repo."
+  default_value: "origin"
+- name: "S2I_FORK_ID"
+  description: ""
+  default_value: "openshift"
+- name: "S2I_BRANCH"
+  description: ""
+  default_value: "master"
+- name: "S2I_REPO"
+  description: "Note: s2i fork amis are not currently supported until we drop golang 1.4 support.  Build an s2i binary manually and give it to test."
+  default_value: "source-to-image"
+- name: "RUN_EXTENDED"
+  description: "In addition to the verification tests include the extended tests.  Note:  The ami will still be marked qe-ready and the qe tests launched even if these tests fail."
+  default_value: "false"
+- name: "RUN_EXTENDED"
+  description: "Run the QE 'smoke' tests after build and basic verification success.  These tests should be run for most amis."
+  default_value: "true"
+- name: "RUN_QE_ACCEPTANCE_TESTS"
+  description: "Run the QE 'acceptance' tests after build and basic verification success."
+  default_value: "false"
+- name: "RUN_QE_FULLFUNCTIONAL_TESTS"
+  description: "Run the QE 'fullfunction' tests after build and basic verification success."
+  default_value: "false"
+- name: "TARGET_ORIGIN_BRANCH"
+  description: "The target branch for your pull request. For instance, if you are writing a feature for OSE 3.3, this should be 'enterprise-3.3'."
+  default_value: "master"
+provision:
+  os: "rhel"
+  stage: "bare"
+  provider: "aws"
+actions:
+  - type: "host_script"
+    title: "install CI user account"
+    script: |-
+      oct prepare user
+      sed -i 's/User ec2-user/User origin/g' ./.config/origin-ci-tool/inventory/.ssh_config
+  - type: "host_script"
+    title: "install base dependencies"
+    script: |-
+      oct prepare dependencies
+  - type: "host_script"
+    title: "prepare environment"
+    script: |-
+      set +o nounset
```

Comment:
This was breaking in `TRELLO_URL`. The thing is that bash shouldn't break there because there is a non-empty check. I think this may be related to the fact that `TRELLO_URL` is the only parameter that doesn't get a default value so it may not be exported in this script at all?